### PR TITLE
Redirection error fix(NSA-6607)

### PR DESCRIPTION
**Pull request by rashthedude**

### CODE REVIEW

File: src/app/register/views/verify.ejs [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+<% if (locals.duplicateEntry) { %>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <h1 class="govuk-heading-xl">
+                There has been an error
+            </h1>
+            <p class="govuk-body">There is an error with this email address. Contact the support desk for help.</p>
+        </div>
+    </div>
+<% } else if (locals.email && !locals.showResendLink && !locals.duplicateEntry) { %>
```
Comment (ipredoi) [inline src/app/register/views/verify.ejs:10]: Can we replace `else if` statement with `else` statement - by using `else if` the page will lose the form to insert the verify code and will display only the green and the blue banner

File: src/app/register/views/verify.ejs [hunk 2]
```diff
@@ -50,7 +59,7 @@
         </p>
     </div>
     </div>
-<% } %>
+<% } else { %>
```
Comment (ipredoi) [inline src/app/register/views/verify.ejs:62]: Can we remove the `else` statement to display the verification code form when the email is valid?

**Review by ipredoi — CHANGES_REQUESTED** [review]
Can we please change the else and else if statements to bring back the Confirm email address form?
image

Can you please check if npm  version is `6.14.15` when installing packages?

**Review by malikimran34 — APPROVED** [review]

File: src/app/register/views/verify.ejs (hunk 2 above)
Comment (rashthedude) [reply src/app/register/views/verify.ejs:62]: See above comment.

File: src/app/register/views/verify.ejs (hunk 1 above)
Comment (rashthedude) [reply src/app/register/views/verify.ejs]: Logic further simplified. Can you have a look at it again please @ipredoi ?

**Review by ipredoi — APPROVED** [review]
👍

File: src/app/register/views/verify.ejs (hunk 1 above)
Comment (ipredoi) [reply src/app/register/views/verify.ejs]: Looks good now. Thanks Rashid

**Pull request merged**